lib/vmpooler/pool_manager.rb

```diff
@@ -555,6 +581,17 @@ def _check_pool(pool)
         end
       end
 
+      # MIGRATIONS
+      $redis.smembers('vmpooler__migrating__' + pool['name']).each do |vm|
+        if inventory[vm]
+          begin
+            migrate_vm(vm, pool['name'])
+          rescue
```

Comment:
It might be handy for logging purposes here to make this:

``` ruby
begin
  migrate_vm(vm, pool['name'])
rescue => err
  $logger.log('s', '[x] [' + pool['name'] + "] '" + vm + "' failed to migrate: #{err}")
end
```

The log line could also be written as:

``` ruby
$logger.log('s', "[x] [#{pool['name']}] '#{vm}' failed to migrate: #{err}")
```